src/components/shared/Camera.tsx

```diff
@@ -54,7 +54,8 @@ class Camera extends React.Component<CameraProps, CameraState> {
   startRecordVideo = (camera: CameraObject) => {
     camera
       .recordAsync({
-        // quality: Camera.Constants.VideoQuality['720p'],
+        // @ts-ignore
```

Comment:
`// @ts-ignore: Expo typings are missing VideoQuality`

and to go the extra mile drop a link to the relevant typing PR